frontend/src/actions/ilabActions.js

```diff
@@ -75,7 +75,7 @@ export const applyFilters = () => (dispatch) => {
   dispatch(setIlabOffset(INITAL_OFFSET));
   dispatch(setIlabPage(START_PAGE));
   dispatch(fetchIlabJobs());
-  dispatch(buildFilterData());
+  //  dispatch(buildFilterData());
```

Comment:
Let's not leave commented code. (I guess this snuck in during all my rebases ... 😦 )

Reply:
Got it!